varnish/tests/conftest.py

```diff
@@ -27,7 +27,11 @@ def check():
 
 @pytest.fixture(scope='session')
 def dd_environment():
-    compose_file = os.path.join(common.HERE, 'compose', 'docker-compose.yaml')
+    varnish_version = os.getenv("VARNISH_VERSION")
+    if varnish_version.startswith("5"):
+        compose_file = os.path.join(common.HERE, 'compose', 'docker-compose-old.yaml')
```

Comment:
(Nit) Naming the file after the varnish version it targets ("old" is not very explicit):

```suggestion
        compose_file = os.path.join(common.HERE, 'compose', 'docker-compose-5.yaml')
```